main.js

```diff
@@ -173,6 +178,8 @@ var start_local_server = function() {
       serverRunning = false;
     });
     subpy.unref();
+  } else {
+    mainWindow.webContents.executeJavaScript("console.log('Unable to find OpenBazaar-Server at: '" + serverPath + "')");
```

Comment:
I don't believe there is anything that calls it before it's instantiated timing wise but I will move it anyways.